### PR TITLE
Create Installation Guide for Django and Flask

### PR DESCRIPTION
**Pull request by alayek**
Creating a wiki article entry, for the installation and set-up process of Django and Flask framework.

### CODE REVIEW

**Comment by raisedadead** [comment]
@alayek Ignore the bot, we are testing it.
Also can you please edit the PR description, seems you forgot to remove the template?

**Comment by raisedadead** [comment]
@Rafase282 Apart from the overall QA, need someone to review the content. Article looks pretty solid to me, just want to have a third eye to look at this.

**Comment by alayek** [comment]
Thanks @raisedadead. Yes, please have someone review it.

**Comment by QuincyLarson** [comment]
@raisedadead @alayek @Rafase282 Thanks for making sure this gets merged ASAP. We will also want to push this to http://github.com/freecodecamp/freecodecamp/wiki since freecodecamp.com's wiki is still a bit buggy.

This Python guide will be a big part of an article I am writing for publication Wednesday morning about the Wiki's role in our expansion into covering additional back end languages.

**Comment by alayek** [comment]
Thanks @QuincyLarson @Rafase282 @raisedadead

File: Install-Django-Flask.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+```python
+print("Hello World")
+```
+
+Both Python 2 and Python 3 are cool. Most Linux and OSX distributions come preinstalled with Python 2 as the default Python. And Python 3 was born out of the insatiable quest of more readable, more beautiful language construct.
+
+So, the only question you need to concern yourself with right now; is which one you should pick. Yes, it takes time to even get familiar with most common aspects with either Python; and your time is important. So, invest your time and effort wisely!
+
+If you are new to Python; you should pick Python 3. While Python 2 is well-supported and popular; most common libraries and frameworks in Python prefer Python 3. Django officially [recommends](https://docs.djangoproject.com/en/1.9/faq/install/#faq-python-version-support) Python 3. Flask and all its dependencies are also [supported](http://flask.pocoo.org/docs/0.10/python3/#python3-support) on Python 3.
```
Comment (raisedadead) [inline Install-Django-Flask.md]: Add official links to Django and Flask.

File: Install-Django-Flask.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+This article would use Python 3 to set up the web-frameworks in your development environment. But before that, let's ensure we have Python 3 and we know how to use it!
+
+### Installing and Using Python 3
+
+You can download Python from this official [link](https://www.python.org/downloads/). Based on your OS (Windows or Linux or OSX), you might want to install Python 3 following [these instructions](http://docs.python-guide.org/en/latest/starting/installation/).
+
+It is always a great idea to [sandbox](https://en.wikipedia.org/wiki/Sandbox_(computer_security)) your Python installation; and keepig it separate from your _System Python_. The _System Python_ is the path to Python interpreter, which is used by other modules installed along with your OS.
```
Comment (raisedadead) [inline Install-Django-Flask.md]: Typo `keepig > keeping`

File: Install-Django-Flask.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+### Wrapping Up
+
+We have successfully installed the web-framework of our need. However, it's not yet complete. Most web applications are content and data driven - so we need a data storage. Or, a Database, if you will.
+
+In next article, we would be discussing how to install PostgreSQL and use it with our Python web application.
+
+A point to ponder - we have been using `pip` heavily, but we have barely said anything about it. Well, for now, it's just a package manager like `npm`. It has some differences with `npm`; but, you don't need to worry about that now. If you are interested, do check out the [official `pip` documentation](http://pip-python3.readthedocs.org/en/latest/index.html).
+
+If you have suggestions or questions, come join us on [gitter](https://gitter.im/FreeCodeCamp/FreeCodeCamp).
```
Comment (raisedadead) [inline Install-Django-Flask.md]: _If you have suggestions or questions, come join us on [gitter](https://gitter.im/FreeCodeCamp/FreeCodeCamp)_.

**Pull request merged**